src/CmsTextFieldBundle/objects/TCMSTextFieldEndPoint.class.php

```diff
@@ -1232,4 +1232,9 @@ protected function isForceThumbnailGenerationOnFullSizeImagesEnabled()
     {
         return false;
     }
+
+    private function doesAllowScriptTags(): bool
```

Comment:
Why is this method called `doesAllowScriptTags` instead of `allowScriptTags`? This is somewhat unconventional.

Reply:
`allowScriptTags` would (in code context as opposed to the config parameter name) imply an action that makes script takes allowed - if possible methods returning booleans should have an `is` or `has` prefix. I changed the method name to `isScriptTagAllowed` as the plural isn't really required.

Reply:
perfect